src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensions.scala

```diff
@@ -356,6 +356,7 @@ class GoogleExtensions(
         // SA does not exist in google, create it and add it to the proxy group
         case None =>
           for {
+            _ <- assertProjectIsActive(project)
```

Comment:
should the call to `assertProjectInTerraOrg` come first?

Reply:
Good call, it makes sense to check if it's a Terra project first